### DISCUSSION TITLE
antd 和 tailwind 同时使用，打开控制台调试样式一直闪烁

### DISCUSSION THREAD

**Discussion by lolocoo** [web-infra-dev/rsbuild#2236] [Q&A]
antd 和 tailwind 同时使用，打开控制台调试样式一直闪烁，一直在重新加载样式，有时 tailwind 还加载不到样式导致无法调适。关闭控制台该现象消失。

**Comment by lolocoo** [comment]
打开浏览器控制台时候，发现tailwind 的样式没有加载，关闭刷新恢复正常。

**Reply by chenjiahan** [reply]
之前同事有过类似的问题，是 Chrome Canary 版本导致的，换成 Chrome 稳定版本后正常

**Reply by lolocoo** [reply]
我的版本就是正式版的。
Chrome is up to date
Version 124.0.6367.93 (Official Build) (arm64)

**Reply by chenjiahan** [reply]
可以提供复现仓库看看

**Reply by lolocoo** [reply]
https://github.com/lolocoo/rsbuild-scaffold.git
🙏

**Comment by lolocoo** [comment]
这个问题有解决方案么，开发起来非常痛苦

**Reply by chenjiahan** [reply]
我用上述的 demo 未能复现你描述的问题哈，应该是本地浏览器环境的问题，不像是 Rsbuild 导致的。

![image](https://github.com/web-infra-dev/rsbuild/assets/7237365/a723f7f3-fe93-40ad-a643-5e3a875e2361)

**Reply by lolocoo** [reply]
都是正常的版本

Version 124.0.6367.119 (Official Build) (arm64)

![20240506-172527](https://github.com/web-infra-dev/rsbuild/assets/25301/5139453d-0b20-47fa-9c3d-4fe9973708f6)

**Reply by 9aoy** [reply]
根据你给的复现仓库无法复现问题 +1。复现仓库和你截图里有一些出入：
- ‘antd 和 tailwind 同时使用’：demo 里好像并没有使用 antd
- 截图里的 CSS 样式看起来是插入到 DOM 中的，但是 demo 里是提取到独立的 .css 文件(rsbuild 默认)

另外，可以根据以下步骤进行自查：
- 尝试使用其他浏览器打开
- 该问题是否只在 dev / build 下出现，通过关闭 dev.hmr / dev.liveReload 可排除是否是热更新问题

**Reply by xc2** [reply]
看起来像是浏览器插件导致的，隐身模式试一下？
